### DISCUSSION TITLE
API source

### DISCUSSION THREAD

**Discussion by Mojzi2** [HC200ok/vue3-easy-data-table#134] [Q&A]
Hi,
How can I use an API for data source?
For example I tried to display the data from https://jsonplaceholder.typicode.com/users but failed.
Can you write a code example for this?
Thank you!

**Comment by HC200ok** [comment]
@Mojzi2 Firstly, you should implement the restful API if it does not exist, then refer to the `server-side` mode example here: https://hc200ok.github.io/vue3-easy-data-table-doc/features/server-side-paginate-and-sort.html
